testutils/lint: TestNightlyLint failed

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19100007?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19100007?buildTab=artifacts#/) on master @ [ccc3f31caec8193d217ce6866dfb3757c1ca47f6](https://github.com/cockroachdb/cockroach/commits/ccc3f31caec8193d217ce6866dfb3757c1ca47f6):


```
             1963:  https://www.cockroachlabs.com/docs/v25.2/alter-view.html
        https://www.cockroachlabs.com/docs/v25.2/alter-database.html : 404 Not Found
             772:  https://www.cockroachlabs.com/docs/v25.2/alter-database.html
        https://www.cockroachlabs.com/docs/v25.2/table-expressions.html : 404 Not Found
             487:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             673:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1332:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1523:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1579:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1804:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             2065:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v25.2/alter-table.html : 404 Not Found
             979:  https://www.cockroachlabs.com/docs/v25.2/alter-table.html
        https://www.cockroachlabs.com/docs/v25.2/drop-function.html : 404 Not Found
             1638:  https://www.cockroachlabs.com/docs/v25.2/drop-function.html
        https://www.cockroachlabs.com/docs/v25.2/show-index.html : 404 Not Found
             1251:  https://www.cockroachlabs.com/docs/v25.2/show-index.html
        https://www.cockroachlabs.com/docs/v25.2/show-databases.html : 404 Not Found
             1694:  https://www.cockroachlabs.com/docs/v25.2/show-databases.html
        https://www.cockroachlabs.com/docs/v25.2/savepoint.html : 404 Not Found
             430:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             2348:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        https://www.cockroachlabs.com/docs/v25.2/show-columns.html : 404 Not Found
             1458:  https://www.cockroachlabs.com/docs/v25.2/show-columns.html
        https://www.cockroachlabs.com/docs/v25.2/create-table-as.html : 404 Not Found
             1528:https://www.cockroachlabs.com/docs/v25.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v25.2/show-vars.html : 404 Not Found
             1014:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
             1127:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
        https://www.cockroachlabs.com/docs/v25.2/alter-partition.html : 404 Not Found
             1199:  https://www.cockroachlabs.com/docs/v25.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v25.2/show-create.html : 404 Not Found
             839:  https://www.cockroachlabs.com/docs/v25.2/show-create.html
        https://www.cockroachlabs.com/docs/v25.2/show-default-privileges : 404 Not Found
             1043:  https://www.cockroachlabs.com/docs/v25.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v25.2/select-clause.html : 404 Not Found
             1580:  https://www.cockroachlabs.com/docs/v25.2/select-clause.html
        https://www.cockroachlabs.com/docs/v25.2/import-into.html : 404 Not Found
             798:  https://www.cockroachlabs.com/docs/v25.2/import-into.html
        https://www.cockroachlabs.com/docs/v25.2/grant.html : 404 Not Found
             320:  https://www.cockroachlabs.com/docs/v25.2/grant.html
        https://www.cockroachlabs.com/docs/v25.2/drop-index.html : 404 Not Found
             174:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
             1992:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
        https://www.cockroachlabs.com/docs/v25.2/alter-index.html : 404 Not Found
             522:  https://www.cockroachlabs.com/docs/v25.2/alter-index.html
        https://www.cockroachlabs.com/docs/v25.2/set-transaction.html : 404 Not Found
             1118:https://www.cockroachlabs.com/docs/v25.2/set-transaction.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (1.16s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-48562

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19109400?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19109400?buildTab=artifacts#/) on master @ [589b9ba45190130715ee3ab74d3379a3ff3c81c6](https://github.com/cockroachdb/cockroach/commits/589b9ba45190130715ee3ab74d3379a3ff3c81c6):


```
             1330:  https://www.cockroachlabs.com/docs/v25.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html : 404 Not Found
             1962:  https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/delete.html : 404 Not Found
             44:  https://www.cockroachlabs.com/docs/v25.2/delete.html
        https://www.cockroachlabs.com/docs/v25.2/drop-index.html : 404 Not Found
             534:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
             938:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
        https://www.cockroachlabs.com/docs/v25.2/savepoint.html : 404 Not Found
             154:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             1058:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        https://www.cockroachlabs.com/docs/v25.2/upsert.html : 404 Not Found
             600:  https://www.cockroachlabs.com/docs/v25.2/upsert.html
        https://www.cockroachlabs.com/docs/v25.2/show-databases.html : 404 Not Found
             2183:  https://www.cockroachlabs.com/docs/v25.2/show-databases.html
        https://www.cockroachlabs.com/docs/v25.2/show-zone-configurations.html : 404 Not Found
             11:  https://www.cockroachlabs.com/docs/v25.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v25.2/show-index.html : 404 Not Found
             2021:  https://www.cockroachlabs.com/docs/v25.2/show-index.html
        https://www.cockroachlabs.com/docs/v25.2/alter-table.html : 404 Not Found
             1690:  https://www.cockroachlabs.com/docs/v25.2/alter-table.html
        https://www.cockroachlabs.com/docs/v25.2/alter-type.html : 404 Not Found
             1477:  https://www.cockroachlabs.com/docs/v25.2/alter-type.html
        https://www.cockroachlabs.com/docs/v25.2/update.html : 404 Not Found
             1803:  https://www.cockroachlabs.com/docs/v25.2/update.html
        https://www.cockroachlabs.com/docs/v25.2/create-table-as.html : 404 Not Found
             1946:https://www.cockroachlabs.com/docs/v25.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v25.2/show-default-privileges : 404 Not Found
             1356:  https://www.cockroachlabs.com/docs/v25.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v25.2/alter-database.html : 404 Not Found
             929:  https://www.cockroachlabs.com/docs/v25.2/alter-database.html
        https://www.cockroachlabs.com/docs/v25.2/show-vars.html : 404 Not Found
             276:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
             1219:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
        https://www.cockroachlabs.com/docs/v25.2/show-backup.html : 404 Not Found
             1726:  https://www.cockroachlabs.com/docs/v25.2/show-backup.html
        https://www.cockroachlabs.com/docs/v25.2/alter-procedure.html : 404 Not Found
             2086:  https://www.cockroachlabs.com/docs/v25.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/drop-table.html : 404 Not Found
             2164:  https://www.cockroachlabs.com/docs/v25.2/drop-table.html
        https://www.cockroachlabs.com/docs/v25.2/show-constraints.html : 404 Not Found
             1241:  https://www.cockroachlabs.com/docs/v25.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v25.2/drop-database.html : 404 Not Found
             2247:  https://www.cockroachlabs.com/docs/v25.2/drop-database.html
        https://www.cockroachlabs.com/docs/v25.2/show-partitions.html : 404 Not Found
             285:  https://www.cockroachlabs.com/docs/v25.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v25.2/alter-partition.html : 404 Not Found
             1282:  https://www.cockroachlabs.com/docs/v25.2/alter-partition.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.73s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19112851?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19112851?buildTab=artifacts#/) on master @ [b53856cc8cebf5fe28a8fca3940f5603b5b2d833](https://github.com/cockroachdb/cockroach/commits/b53856cc8cebf5fe28a8fca3940f5603b5b2d833):


```
             1163:  https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/delete.html : 404 Not Found
             1027:  https://www.cockroachlabs.com/docs/v25.2/delete.html
        https://www.cockroachlabs.com/docs/v25.2/truncate.html : 404 Not Found
             1979:  https://www.cockroachlabs.com/docs/v25.2/truncate.html
        https://www.cockroachlabs.com/docs/v25.2/show-backup.html : 404 Not Found
             1846:  https://www.cockroachlabs.com/docs/v25.2/show-backup.html
        https://www.cockroachlabs.com/docs/v25.2/show-create.html : 404 Not Found
             1919:  https://www.cockroachlabs.com/docs/v25.2/show-create.html
        https://www.cockroachlabs.com/docs/v25.2/upsert.html : 404 Not Found
             792:  https://www.cockroachlabs.com/docs/v25.2/upsert.html
        https://www.cockroachlabs.com/docs/v25.2/explain.html : 404 Not Found
             2243:  https://www.cockroachlabs.com/docs/v25.2/explain.html
        https://www.cockroachlabs.com/docs/v25.2/savepoint.html : 404 Not Found
             1063:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             1668:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        https://www.cockroachlabs.com/docs/v25.2/show-grants.html : 404 Not Found
             810:  https://www.cockroachlabs.com/docs/v25.2/show-grants.html
        https://www.cockroachlabs.com/docs/v25.2/set-vars.html : 404 Not Found
             35:  https://www.cockroachlabs.com/docs/v25.2/set-vars.html
             861:https://www.cockroachlabs.com/docs/v25.2/set-vars.html
             1504:https://www.cockroachlabs.com/docs/v25.2/set-vars.html
             1758:  https://www.cockroachlabs.com/docs/v25.2/set-vars.html
        https://www.cockroachlabs.com/docs/v25.2/alter-type.html : 404 Not Found
             1086:  https://www.cockroachlabs.com/docs/v25.2/alter-type.html
        https://www.cockroachlabs.com/docs/v25.2/restore.html : 404 Not Found
             1218:  https://www.cockroachlabs.com/docs/v25.2/restore.html
        https://www.cockroachlabs.com/docs/v25.2/drop-table.html : 404 Not Found
             1096:  https://www.cockroachlabs.com/docs/v25.2/drop-table.html
        https://www.cockroachlabs.com/docs/v25.2/show-ranges.html : 404 Not Found
             75:  https://www.cockroachlabs.com/docs/v25.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v25.2/show-vars.html : 404 Not Found
             232:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
             1265:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
        https://www.cockroachlabs.com/docs/v25.2/show-constraints.html : 404 Not Found
             1568:  https://www.cockroachlabs.com/docs/v25.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v25.2/insert.html : 404 Not Found
             733:  https://www.cockroachlabs.com/docs/v25.2/insert.html
        https://www.cockroachlabs.com/docs/v25.2/grant.html : 404 Not Found
             2386:  https://www.cockroachlabs.com/docs/v25.2/grant.html
        https://www.cockroachlabs.com/docs/v25.2/create-index.html : 404 Not Found
             771:https://www.cockroachlabs.com/docs/v25.2/create-index.html
        https://www.cockroachlabs.com/docs/v25.2/backup.html : 404 Not Found
             428:  https://www.cockroachlabs.com/docs/v25.2/backup.html
        https://www.cockroachlabs.com/docs/v25.2/show-columns.html : 404 Not Found
             2265:  https://www.cockroachlabs.com/docs/v25.2/show-columns.html
        https://www.cockroachlabs.com/docs/v25.2/show-default-privileges : 404 Not Found
             1558:  https://www.cockroachlabs.com/docs/v25.2/show-default-privileges
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19115810?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19115810?buildTab=artifacts#/) on master @ [b53856cc8cebf5fe28a8fca3940f5603b5b2d833](https://github.com/cockroachdb/cockroach/commits/b53856cc8cebf5fe28a8fca3940f5603b5b2d833):


```
             2403:  https://www.cockroachlabs.com/docs/v25.2/truncate.html
        https://www.cockroachlabs.com/docs/v25.2/drop-function.html : 404 Not Found
             1869:  https://www.cockroachlabs.com/docs/v25.2/drop-function.html
        https://www.cockroachlabs.com/docs/v25.2/alter-type.html : 404 Not Found
             1409:  https://www.cockroachlabs.com/docs/v25.2/alter-type.html
        https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html : 404 Not Found
             1251:  https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/delete.html : 404 Not Found
             1648:  https://www.cockroachlabs.com/docs/v25.2/delete.html
        https://www.cockroachlabs.com/docs/v25.2/update.html : 404 Not Found
             2149:  https://www.cockroachlabs.com/docs/v25.2/update.html
        https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html : 404 Not Found
             227:  https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/drop-index.html : 404 Not Found
             1167:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
             1212:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
        https://www.cockroachlabs.com/docs/v25.2/insert.html : 404 Not Found
             1202:  https://www.cockroachlabs.com/docs/v25.2/insert.html
        https://www.cockroachlabs.com/docs/v25.2/create-view.html : 404 Not Found
             1480:  https://www.cockroachlabs.com/docs/v25.2/create-view.html
        https://www.cockroachlabs.com/docs/v25.2/table-expressions.html : 404 Not Found
             71:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             298:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             639:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1477:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1843:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             2145:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             2161:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v25.2/show-users.html : 404 Not Found
             599:  https://www.cockroachlabs.com/docs/v25.2/show-users.html
        https://www.cockroachlabs.com/docs/v25.2/revoke.html : 404 Not Found
             1574:  https://www.cockroachlabs.com/docs/v25.2/revoke.html
        https://www.cockroachlabs.com/docs/v25.2/show-databases.html : 404 Not Found
             695:  https://www.cockroachlabs.com/docs/v25.2/show-databases.html
        https://www.cockroachlabs.com/docs/v25.2/savepoint.html : 404 Not Found
             518:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             2027:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html : 404 Not Found
             1104:  https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html
             1117:https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v25.2/alter-database.html : 404 Not Found
             861:  https://www.cockroachlabs.com/docs/v25.2/alter-database.html
        https://www.cockroachlabs.com/docs/v25.2/alter-table.html : 404 Not Found
             841:  https://www.cockroachlabs.com/docs/v25.2/alter-table.html
        https://www.cockroachlabs.com/docs/v25.2/alter-partition.html : 404 Not Found
             1534:  https://www.cockroachlabs.com/docs/v25.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v25.2/restore.html : 404 Not Found
             200:  https://www.cockroachlabs.com/docs/v25.2/restore.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.57s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19129011?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19129011?buildTab=artifacts#/) on master @ [2f369235e35a15efcaf0f52aec15db442b35306d](https://github.com/cockroachdb/cockroach/commits/2f369235e35a15efcaf0f52aec15db442b35306d):


```
             2181:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v25.2/show-ranges.html : 404 Not Found
             1584:  https://www.cockroachlabs.com/docs/v25.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v25.2/create-table.html : 404 Not Found
             1916:https://www.cockroachlabs.com/docs/v25.2/create-table.html
        https://www.cockroachlabs.com/docs/v25.2/alter-type.html : 404 Not Found
             740:  https://www.cockroachlabs.com/docs/v25.2/alter-type.html
        https://www.cockroachlabs.com/docs/v25.2/show-backup.html : 404 Not Found
             372:  https://www.cockroachlabs.com/docs/v25.2/show-backup.html
        https://www.cockroachlabs.com/docs/v25.2/show-partitions.html : 404 Not Found
             871:  https://www.cockroachlabs.com/docs/v25.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v25.2/show-users.html : 404 Not Found
             1137:  https://www.cockroachlabs.com/docs/v25.2/show-users.html
        https://www.cockroachlabs.com/docs/v25.2/restore.html : 404 Not Found
             1469:  https://www.cockroachlabs.com/docs/v25.2/restore.html
        https://www.cockroachlabs.com/docs/v25.2/show-index.html : 404 Not Found
             1560:  https://www.cockroachlabs.com/docs/v25.2/show-index.html
        https://www.cockroachlabs.com/docs/v25.2/create-index.html : 404 Not Found
             1014:https://www.cockroachlabs.com/docs/v25.2/create-index.html
        https://www.cockroachlabs.com/docs/v25.2/revoke.html : 404 Not Found
             249:  https://www.cockroachlabs.com/docs/v25.2/revoke.html
        https://www.cockroachlabs.com/docs/v25.2/show-vars.html : 404 Not Found
             91:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
             2098:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
        https://www.cockroachlabs.com/docs/v25.2/alter-database.html : 404 Not Found
             1937:  https://www.cockroachlabs.com/docs/v25.2/alter-database.html
        https://www.cockroachlabs.com/docs/v25.2/show-columns.html : 404 Not Found
             259:  https://www.cockroachlabs.com/docs/v25.2/show-columns.html
        https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html : 404 Not Found
             1503:  https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/alter-procedure.html : 404 Not Found
             961:  https://www.cockroachlabs.com/docs/v25.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/drop-database.html : 404 Not Found
             988:  https://www.cockroachlabs.com/docs/v25.2/drop-database.html
        https://www.cockroachlabs.com/docs/v25.2/show-default-privileges : 404 Not Found
             1815:  https://www.cockroachlabs.com/docs/v25.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v25.2/alter-index.html : 404 Not Found
             654:  https://www.cockroachlabs.com/docs/v25.2/alter-index.html
        https://www.cockroachlabs.com/docs/v25.2/alter-table.html : 404 Not Found
             561:  https://www.cockroachlabs.com/docs/v25.2/alter-table.html
        https://www.cockroachlabs.com/docs/v25.2/create-database.html : 404 Not Found
             979:  https://www.cockroachlabs.com/docs/v25.2/create-database.html
        https://www.cockroachlabs.com/docs/v25.2/create-table-as.html : 404 Not Found
             1917:https://www.cockroachlabs.com/docs/v25.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v25.2/show-constraints.html : 404 Not Found
             1485:  https://www.cockroachlabs.com/docs/v25.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v25.2/import-into.html : 404 Not Found
             143:  https://www.cockroachlabs.com/docs/v25.2/import-into.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.58s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19141762?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19141762?buildTab=artifacts#/) on master @ [3227e31cc49650fae510626b79a2e93614195bc6](https://github.com/cockroachdb/cockroach/commits/3227e31cc49650fae510626b79a2e93614195bc6):


```
             488:https://www.cockroachlabs.com/docs/v25.2/create-index.html
        https://www.cockroachlabs.com/docs/v25.2/alter-type.html : 404 Not Found
             908:  https://www.cockroachlabs.com/docs/v25.2/alter-type.html
        https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html : 404 Not Found
             781:  https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/show-default-privileges : 404 Not Found
             568:  https://www.cockroachlabs.com/docs/v25.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v25.2/alter-view.html : 404 Not Found
             2338:  https://www.cockroachlabs.com/docs/v25.2/alter-view.html
        https://www.cockroachlabs.com/docs/v25.2/insert.html : 404 Not Found
             885:  https://www.cockroachlabs.com/docs/v25.2/insert.html
        https://www.cockroachlabs.com/docs/v25.2/alter-database.html : 404 Not Found
             146:  https://www.cockroachlabs.com/docs/v25.2/alter-database.html
        https://www.cockroachlabs.com/docs/v25.2/show-tables.html : 404 Not Found
             227:  https://www.cockroachlabs.com/docs/v25.2/show-tables.html
        https://www.cockroachlabs.com/docs/v25.2/show-index.html : 404 Not Found
             752:  https://www.cockroachlabs.com/docs/v25.2/show-index.html
        https://www.cockroachlabs.com/docs/v25.2/set-vars.html : 404 Not Found
             279:https://www.cockroachlabs.com/docs/v25.2/set-vars.html
             1118:  https://www.cockroachlabs.com/docs/v25.2/set-vars.html
             1268:https://www.cockroachlabs.com/docs/v25.2/set-vars.html
             1506:  https://www.cockroachlabs.com/docs/v25.2/set-vars.html
        https://www.cockroachlabs.com/docs/v25.2/create-table.html : 404 Not Found
             1767:https://www.cockroachlabs.com/docs/v25.2/create-table.html
        https://www.cockroachlabs.com/docs/v25.2/backup.html : 404 Not Found
             949:  https://www.cockroachlabs.com/docs/v25.2/backup.html
        https://www.cockroachlabs.com/docs/v25.2/create-function.html : 404 Not Found
             1027:  https://www.cockroachlabs.com/docs/v25.2/create-function.html
        https://www.cockroachlabs.com/docs/v25.2/show-databases.html : 404 Not Found
             1645:  https://www.cockroachlabs.com/docs/v25.2/show-databases.html
        https://www.cockroachlabs.com/docs/v25.2/select-clause.html : 404 Not Found
             1944:  https://www.cockroachlabs.com/docs/v25.2/select-clause.html
        https://www.cockroachlabs.com/docs/v25.2/drop-function.html : 404 Not Found
             2397:  https://www.cockroachlabs.com/docs/v25.2/drop-function.html
        https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html : 404 Not Found
             1345:  https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html : 404 Not Found
             65:  https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/show-users.html : 404 Not Found
             2182:  https://www.cockroachlabs.com/docs/v25.2/show-users.html
        https://www.cockroachlabs.com/docs/v25.2/savepoint.html : 404 Not Found
             190:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             1539:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        https://www.cockroachlabs.com/docs/v25.2/update.html : 404 Not Found
             1335:  https://www.cockroachlabs.com/docs/v25.2/update.html
        https://www.cockroachlabs.com/docs/v25.2/show-vars.html : 404 Not Found
             1414:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
             1450:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.56s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19154604?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19154604?buildTab=artifacts#/) on master @ [d1e0f7c844d9bcad23fe96641a689e078fdcdc17](https://github.com/cockroachdb/cockroach/commits/d1e0f7c844d9bcad23fe96641a689e078fdcdc17):


```
        https://www.cockroachlabs.com/docs/v25.2/show-constraints.html : 404 Not Found
             1757:  https://www.cockroachlabs.com/docs/v25.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v25.2/create-table.html : 404 Not Found
             161:https://www.cockroachlabs.com/docs/v25.2/create-table.html
        https://www.cockroachlabs.com/docs/v25.2/table-expressions.html : 404 Not Found
             157:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             371:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             552:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             815:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1057:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1446:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             2199:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v25.2/show-ranges.html : 404 Not Found
             1000:  https://www.cockroachlabs.com/docs/v25.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html : 404 Not Found
             1778:  https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/show-databases.html : 404 Not Found
             2005:  https://www.cockroachlabs.com/docs/v25.2/show-databases.html
        https://www.cockroachlabs.com/docs/v25.2/show-create.html : 404 Not Found
             1495:  https://www.cockroachlabs.com/docs/v25.2/show-create.html
        https://www.cockroachlabs.com/docs/v25.2/set-vars.html : 404 Not Found
             826:  https://www.cockroachlabs.com/docs/v25.2/set-vars.html
             838:  https://www.cockroachlabs.com/docs/v25.2/set-vars.html
             1747:https://www.cockroachlabs.com/docs/v25.2/set-vars.html
             1901:https://www.cockroachlabs.com/docs/v25.2/set-vars.html
        https://www.cockroachlabs.com/docs/v25.2/upsert.html : 404 Not Found
             1523:  https://www.cockroachlabs.com/docs/v25.2/upsert.html
        https://www.cockroachlabs.com/docs/v25.2/drop-table.html : 404 Not Found
             2294:  https://www.cockroachlabs.com/docs/v25.2/drop-table.html
        https://www.cockroachlabs.com/docs/v25.2/create-view.html : 404 Not Found
             374:  https://www.cockroachlabs.com/docs/v25.2/create-view.html
        https://www.cockroachlabs.com/docs/v25.2/show-default-privileges : 404 Not Found
             2219:  https://www.cockroachlabs.com/docs/v25.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v25.2/import-into.html : 404 Not Found
             2284:  https://www.cockroachlabs.com/docs/v25.2/import-into.html
        https://www.cockroachlabs.com/docs/v25.2/show-partitions.html : 404 Not Found
             1546:  https://www.cockroachlabs.com/docs/v25.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html : 404 Not Found
             1355:  https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/create-function.html : 404 Not Found
             700:  https://www.cockroachlabs.com/docs/v25.2/create-function.html
        https://www.cockroachlabs.com/docs/v25.2/savepoint.html : 404 Not Found
             278:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             1045:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        https://www.cockroachlabs.com/docs/v25.2/restore.html : 404 Not Found
             2357:  https://www.cockroachlabs.com/docs/v25.2/restore.html
        https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html : 404 Not Found
             575:  https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.56s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19166563?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19166563?buildTab=artifacts#/) on master @ [8fe88ea2839a27a1475d5a155a3535eed95d9b13](https://github.com/cockroachdb/cockroach/commits/8fe88ea2839a27a1475d5a155a3535eed95d9b13):


```
             455:https://www.cockroachlabs.com/docs/v25.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/alter-procedure.html : 404 Not Found
             1740:  https://www.cockroachlabs.com/docs/v25.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/alter-partition.html : 404 Not Found
             176:  https://www.cockroachlabs.com/docs/v25.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v25.2/drop-function.html : 404 Not Found
             1829:  https://www.cockroachlabs.com/docs/v25.2/drop-function.html
        https://www.cockroachlabs.com/docs/v25.2/create-function.html : 404 Not Found
             1324:  https://www.cockroachlabs.com/docs/v25.2/create-function.html
        https://www.cockroachlabs.com/docs/v25.2/rollback-transaction.html : 404 Not Found
             1412:  https://www.cockroachlabs.com/docs/v25.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/create-index.html : 404 Not Found
             259:https://www.cockroachlabs.com/docs/v25.2/create-index.html
        https://www.cockroachlabs.com/docs/v25.2/upsert.html : 404 Not Found
             1013:  https://www.cockroachlabs.com/docs/v25.2/upsert.html
        https://www.cockroachlabs.com/docs/v25.2/drop-index.html : 404 Not Found
             743:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
             1334:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
        https://www.cockroachlabs.com/docs/v25.2/show-grants.html : 404 Not Found
             609:  https://www.cockroachlabs.com/docs/v25.2/show-grants.html
        https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html : 404 Not Found
             1176:  https://www.cockroachlabs.com/docs/v25.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/delete.html : 404 Not Found
             1452:  https://www.cockroachlabs.com/docs/v25.2/delete.html
        https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html : 404 Not Found
             795:  https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/show-databases.html : 404 Not Found
             1659:  https://www.cockroachlabs.com/docs/v25.2/show-databases.html
        https://www.cockroachlabs.com/docs/v25.2/show-vars.html : 404 Not Found
             992:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
             1245:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
        https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html : 404 Not Found
             972:  https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/alter-view.html : 404 Not Found
             1680:  https://www.cockroachlabs.com/docs/v25.2/alter-view.html
        https://www.cockroachlabs.com/docs/v25.2/drop-table.html : 404 Not Found
             1650:  https://www.cockroachlabs.com/docs/v25.2/drop-table.html
        https://www.cockroachlabs.com/docs/v25.2/alter-function.html : 404 Not Found
             1703:  https://www.cockroachlabs.com/docs/v25.2/alter-function.html
        https://www.cockroachlabs.com/docs/v25.2/insert.html : 404 Not Found
             426:  https://www.cockroachlabs.com/docs/v25.2/insert.html
        https://www.cockroachlabs.com/docs/v25.2/alter-database.html : 404 Not Found
             1265:  https://www.cockroachlabs.com/docs/v25.2/alter-database.html
        https://www.cockroachlabs.com/docs/v25.2/explain.html : 404 Not Found
             2315:  https://www.cockroachlabs.com/docs/v25.2/explain.html
        https://www.cockroachlabs.com/docs/v25.2/savepoint.html : 404 Not Found
             1726:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             1941:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.59s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19178676?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19178676?buildTab=artifacts#/) on master @ [cafa08712017fe44bace4f1b55a81736eee900b0](https://github.com/cockroachdb/cockroach/commits/cafa08712017fe44bace4f1b55a81736eee900b0):


```
        https://www.cockroachlabs.com/docs/v25.2/revoke.html : 404 Not Found
             1601:  https://www.cockroachlabs.com/docs/v25.2/revoke.html
        https://www.cockroachlabs.com/docs/v25.2/table-expressions.html : 404 Not Found
             97:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             469:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             612:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1297:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1385:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1865:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             2403:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v25.2/show-partitions.html : 404 Not Found
             150:  https://www.cockroachlabs.com/docs/v25.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v25.2/create-table.html : 404 Not Found
             1869:https://www.cockroachlabs.com/docs/v25.2/create-table.html
        https://www.cockroachlabs.com/docs/v25.2/show-index.html : 404 Not Found
             1031:  https://www.cockroachlabs.com/docs/v25.2/show-index.html
        https://www.cockroachlabs.com/docs/v25.2/show-databases.html : 404 Not Found
             794:  https://www.cockroachlabs.com/docs/v25.2/show-databases.html
        https://www.cockroachlabs.com/docs/v25.2/drop-index.html : 404 Not Found
             1645:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
             1902:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
        https://www.cockroachlabs.com/docs/v25.2/set-transaction.html : 404 Not Found
             1939:https://www.cockroachlabs.com/docs/v25.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/truncate.html : 404 Not Found
             300:  https://www.cockroachlabs.com/docs/v25.2/truncate.html
        https://www.cockroachlabs.com/docs/v25.2/delete.html : 404 Not Found
             27:  https://www.cockroachlabs.com/docs/v25.2/delete.html
        https://www.cockroachlabs.com/docs/v25.2/upsert.html : 404 Not Found
             680:  https://www.cockroachlabs.com/docs/v25.2/upsert.html
        https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html : 404 Not Found
             40:https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html
             2107:  https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v25.2/drop-function.html : 404 Not Found
             1636:  https://www.cockroachlabs.com/docs/v25.2/drop-function.html
        https://www.cockroachlabs.com/docs/v25.2/show-columns.html : 404 Not Found
             1800:  https://www.cockroachlabs.com/docs/v25.2/show-columns.html
        https://www.cockroachlabs.com/docs/v25.2/show-tables.html : 404 Not Found
             2116:  https://www.cockroachlabs.com/docs/v25.2/show-tables.html
        https://www.cockroachlabs.com/docs/v25.2/rollback-transaction.html : 404 Not Found
             1334:  https://www.cockroachlabs.com/docs/v25.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/drop-table.html : 404 Not Found
             2137:  https://www.cockroachlabs.com/docs/v25.2/drop-table.html
        https://www.cockroachlabs.com/docs/v25.2/alter-partition.html : 404 Not Found
             1484:  https://www.cockroachlabs.com/docs/v25.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v25.2/backup.html : 404 Not Found
             341:  https://www.cockroachlabs.com/docs/v25.2/backup.html
        https://www.cockroachlabs.com/docs/v25.2/grant.html : 404 Not Found
             2172:  https://www.cockroachlabs.com/docs/v25.2/grant.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.58s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19185030?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19185030?buildTab=artifacts#/) on master @ [c6550afee782a83d06845e6581a6ad17127ff199](https://github.com/cockroachdb/cockroach/commits/c6550afee782a83d06845e6581a6ad17127ff199):


```
        https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html : 404 Not Found
             205:  https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html
             1761:https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v25.2/show-columns.html : 404 Not Found
             1364:  https://www.cockroachlabs.com/docs/v25.2/show-columns.html
        https://www.cockroachlabs.com/docs/v25.2/alter-database.html : 404 Not Found
             98:  https://www.cockroachlabs.com/docs/v25.2/alter-database.html
        https://www.cockroachlabs.com/docs/v25.2/create-function.html : 404 Not Found
             1281:  https://www.cockroachlabs.com/docs/v25.2/create-function.html
        https://www.cockroachlabs.com/docs/v25.2/upsert.html : 404 Not Found
             2355:  https://www.cockroachlabs.com/docs/v25.2/upsert.html
        https://www.cockroachlabs.com/docs/v25.2/create-procedure.html : 404 Not Found
             2049:  https://www.cockroachlabs.com/docs/v25.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/show-grants.html : 404 Not Found
             1223:  https://www.cockroachlabs.com/docs/v25.2/show-grants.html
        https://www.cockroachlabs.com/docs/v25.2/select-clause.html : 404 Not Found
             2298:  https://www.cockroachlabs.com/docs/v25.2/select-clause.html
        https://www.cockroachlabs.com/docs/v25.2/drop-database.html : 404 Not Found
             540:  https://www.cockroachlabs.com/docs/v25.2/drop-database.html
        https://www.cockroachlabs.com/docs/v25.2/create-table.html : 404 Not Found
             921:https://www.cockroachlabs.com/docs/v25.2/create-table.html
        https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html : 404 Not Found
             2205:  https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/show-ranges.html : 404 Not Found
             239:  https://www.cockroachlabs.com/docs/v25.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v25.2/update.html : 404 Not Found
             344:  https://www.cockroachlabs.com/docs/v25.2/update.html
        https://www.cockroachlabs.com/docs/v25.2/drop-index.html : 404 Not Found
             1082:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
             1091:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
        https://www.cockroachlabs.com/docs/v25.2/alter-view.html : 404 Not Found
             255:  https://www.cockroachlabs.com/docs/v25.2/alter-view.html
        https://www.cockroachlabs.com/docs/v25.2/create-database.html : 404 Not Found
             1663:  https://www.cockroachlabs.com/docs/v25.2/create-database.html
        https://www.cockroachlabs.com/docs/v25.2/create-view.html : 404 Not Found
             1177:  https://www.cockroachlabs.com/docs/v25.2/create-view.html
        https://www.cockroachlabs.com/docs/v25.2/show-zone-configurations.html : 404 Not Found
             1578:  https://www.cockroachlabs.com/docs/v25.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v25.2/savepoint.html : 404 Not Found
             57:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             664:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        https://www.cockroachlabs.com/docs/v25.2/revoke.html : 404 Not Found
             195:  https://www.cockroachlabs.com/docs/v25.2/revoke.html
        https://www.cockroachlabs.com/docs/v25.2/drop-function.html : 404 Not Found
             1517:  https://www.cockroachlabs.com/docs/v25.2/drop-function.html
        https://www.cockroachlabs.com/docs/v25.2/show-vars.html : 404 Not Found
             654:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
             1851:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.58s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19190200?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19190200?buildTab=artifacts#/) on master @ [c6550afee782a83d06845e6581a6ad17127ff199](https://github.com/cockroachdb/cockroach/commits/c6550afee782a83d06845e6581a6ad17127ff199):


```
             341:  https://www.cockroachlabs.com/docs/v25.2/create-database.html
        https://www.cockroachlabs.com/docs/v25.2/alter-table.html : 404 Not Found
             1594:  https://www.cockroachlabs.com/docs/v25.2/alter-table.html
        https://www.cockroachlabs.com/docs/v25.2/show-partitions.html : 404 Not Found
             1909:  https://www.cockroachlabs.com/docs/v25.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v25.2/show-tables.html : 404 Not Found
             444:  https://www.cockroachlabs.com/docs/v25.2/show-tables.html
        https://www.cockroachlabs.com/docs/v25.2/truncate.html : 404 Not Found
             1966:  https://www.cockroachlabs.com/docs/v25.2/truncate.html
        https://www.cockroachlabs.com/docs/v25.2/show-vars.html : 404 Not Found
             435:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
             1059:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
        https://www.cockroachlabs.com/docs/v25.2/show-backup.html : 404 Not Found
             1521:  https://www.cockroachlabs.com/docs/v25.2/show-backup.html
        https://www.cockroachlabs.com/docs/v25.2/alter-partition.html : 404 Not Found
             2282:  https://www.cockroachlabs.com/docs/v25.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v25.2/savepoint.html : 404 Not Found
             16:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             1835:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html : 404 Not Found
             55:  https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html
             457:https://www.cockroachlabs.com/docs/v25.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html : 404 Not Found
             1734:  https://www.cockroachlabs.com/docs/v25.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/grant.html : 404 Not Found
             869:  https://www.cockroachlabs.com/docs/v25.2/grant.html
        https://www.cockroachlabs.com/docs/v25.2/create-table.html : 404 Not Found
             2402:https://www.cockroachlabs.com/docs/v25.2/create-table.html
        https://www.cockroachlabs.com/docs/v25.2/table-expressions.html : 404 Not Found
             514:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             666:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             709:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1044:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1761:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             2081:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             2398:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v25.2/show-constraints.html : 404 Not Found
             244:  https://www.cockroachlabs.com/docs/v25.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v25.2/restore.html : 404 Not Found
             2181:  https://www.cockroachlabs.com/docs/v25.2/restore.html
        https://www.cockroachlabs.com/docs/v25.2/show-users.html : 404 Not Found
             1883:  https://www.cockroachlabs.com/docs/v25.2/show-users.html
        https://www.cockroachlabs.com/docs/v25.2/revoke.html : 404 Not Found
             1153:  https://www.cockroachlabs.com/docs/v25.2/revoke.html
        https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html : 404 Not Found
             1792:  https://www.cockroachlabs.com/docs/v25.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/drop-database.html : 404 Not Found
             193:  https://www.cockroachlabs.com/docs/v25.2/drop-database.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.60s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19202838?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19202838?buildTab=artifacts#/) on master @ [61700b1044e25a8ea71e4c07a1681be521312760](https://github.com/cockroachdb/cockroach/commits/61700b1044e25a8ea71e4c07a1681be521312760):


```
             1378:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
             1571:  https://www.cockroachlabs.com/docs/v25.2/savepoint.html
        https://www.cockroachlabs.com/docs/v25.2/alter-procedure.html : 404 Not Found
             2178:  https://www.cockroachlabs.com/docs/v25.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/rollback-transaction.html : 404 Not Found
             576:  https://www.cockroachlabs.com/docs/v25.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v25.2/drop-database.html : 404 Not Found
             2098:  https://www.cockroachlabs.com/docs/v25.2/drop-database.html
        https://www.cockroachlabs.com/docs/v25.2/show-grants.html : 404 Not Found
             1128:  https://www.cockroachlabs.com/docs/v25.2/show-grants.html
        https://www.cockroachlabs.com/docs/v25.2/create-function.html : 404 Not Found
             2393:  https://www.cockroachlabs.com/docs/v25.2/create-function.html
        https://www.cockroachlabs.com/docs/v25.2/alter-partition.html : 404 Not Found
             806:  https://www.cockroachlabs.com/docs/v25.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v25.2/table-expressions.html : 404 Not Found
             78:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             384:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             504:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             957:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             1471:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             2269:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
             2374:  https://www.cockroachlabs.com/docs/v25.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v25.2/drop-index.html : 404 Not Found
             170:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
             294:  https://www.cockroachlabs.com/docs/v25.2/drop-index.html
        https://www.cockroachlabs.com/docs/v25.2/insert.html : 404 Not Found
             987:  https://www.cockroachlabs.com/docs/v25.2/insert.html
        https://www.cockroachlabs.com/docs/v25.2/show-zone-configurations.html : 404 Not Found
             1038:  https://www.cockroachlabs.com/docs/v25.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v25.2/create-table-as.html : 404 Not Found
             962:https://www.cockroachlabs.com/docs/v25.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v25.2/show-columns.html : 404 Not Found
             1846:  https://www.cockroachlabs.com/docs/v25.2/show-columns.html
        https://www.cockroachlabs.com/docs/v25.2/alter-table.html : 404 Not Found
             448:  https://www.cockroachlabs.com/docs/v25.2/alter-table.html
        https://www.cockroachlabs.com/docs/v25.2/grant.html : 404 Not Found
             42:  https://www.cockroachlabs.com/docs/v25.2/grant.html
        https://www.cockroachlabs.com/docs/v25.2/create-procedure.html : 404 Not Found
             646:  https://www.cockroachlabs.com/docs/v25.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v25.2/explain.html : 404 Not Found
             2013:  https://www.cockroachlabs.com/docs/v25.2/explain.html
        https://www.cockroachlabs.com/docs/v25.2/show-vars.html : 404 Not Found
             1116:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
             1418:  https://www.cockroachlabs.com/docs/v25.2/show-vars.html
        https://www.cockroachlabs.com/docs/v25.2/alter-function.html : 404 Not Found
             1494:  https://www.cockroachlabs.com/docs/v25.2/alter-function.html
        https://www.cockroachlabs.com/docs/v25.2/show-databases.html : 404 Not Found
             1002:  https://www.cockroachlabs.com/docs/v25.2/show-databases.html
        54 errors
--- FAIL: TestNightlyLint/TestHelpURLs (0.61s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/build/info.go
+++ b/pkg/build/info.go
@@ -225,3 +225,8 @@ func TestingOverrideVersion(v string) func() {
 func MakeIssueURL(issue int) string {
 	return fmt.Sprintf("https://go.crdb.dev/issue-v/%d/%s", issue, VersionForURLs())
 }
+
+// ParsedVersion returns the parsed version.txt.
+func ParsedVersion() *version.Version {
+	return parsedVersionTxt
+}

--- a/pkg/testutils/lint/BUILD.bazel
+++ b/pkg/testutils/lint/BUILD.bazel
@@ -35,6 +35,7 @@ go_test(
     ],
     visibility = ["//build/bazelutil:__subpackages__"],
     deps = [
+        "//pkg/build",
         "//pkg/build/bazel",
         "//pkg/cmd/urlcheck/lib/urlcheck",
         "//pkg/internal/codeowners",

--- a/pkg/testutils/lint/nightly_lint_test.go
+++ b/pkg/testutils/lint/nightly_lint_test.go
@@ -13,6 +13,7 @@ import (
 	"os/exec"
 	"testing"
 
+	"github.com/cockroachdb/cockroach/pkg/build"
 	"github.com/cockroachdb/cockroach/pkg/cmd/urlcheck/lib/urlcheck"
 	sqlparser "github.com/cockroachdb/cockroach/pkg/sql/parser"
 	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
@@ -24,6 +25,9 @@ func TestNightlyLint(t *testing.T) {
 	// TestHelpURLs checks that all help texts have a valid documentation URL.
 	t.Run("TestHelpURLs", func(t *testing.T) {
 		skip.UnderShort(t)
+		if build.ParsedVersion().IsPrerelease() || build.ParsedVersion().IsCustomOrNightlyBuild() {
+			skip.IgnoreLint(t, "pre-release or customized build")
+		}
 		if pkgSpecified {
 			skip.IgnoreLint(t, "PKG specified")
 		}

